### DISCUSSION TITLE
Worskhop mods not working with DFHack

### DISCUSSION THREAD

**Discussion by Kayrosis** [DFHack/dfhack#2685] [General]
Not sure if this is a bug or not, but when I installed DFHack, when I generate a new world and add workshop mods, such as Armok's Bucket of Blessings, the workshops do not appear in the game menus. They're still present in worlds I made before installing DFHack, but not in newly created ones.

**Comment by myk002** [comment]
DFHack doesn't change anything about workshop mods. We haven't even defined a format yet for workshop mods to add scripts. Is there a way to tell if the mod was successfully "activated" for your newer worlds?
